src/Features/ExternalAccess/HotReload/Api/HotReloadMSBuildWorkspace.ProjectFileInfoProvider.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+extern alias BuildHost;
```

Comment:
Currently needed to avoid ambiguities between contracts and polyfils flowing from BuildHost and Workspace via IVTs.

Will look into using EmbeddedAttribute to avoid.